ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -195,7 +195,6 @@ public Response handleKsqlStatements(
       @Context final ServiceContext serviceContext,
       final KsqlRequest request
   ) {
-    LOG.info("Received: " + request);
```

Comment:
could we introduce a log statement somewhere (say where we check for `isStatic`) that logs the sql statement? That way for non pull queries you still have similar log statement (which i guess would be useful)